Source/Kesmai.Server/Game/Items/Equipment/Armors/LeatherArmor.cs

```diff
@@ -21,11 +29,28 @@ public partial class LeatherArmor : Armor
 		/// <inheritdoc />
 		public override int BashingProtection => 1;
 		
+		public override int BaseArmorBonus // This is now configured to what we set.
+		{
+			get
```

Comment:
I could see this going either way.  we may want to be able to give mobs stronger armor without having to equip them with icies, drakes, etc.  right now we only really have 3 armors for mobs (leather, chain, plate).  I could see all 3 getting this treatment simply to give mobs better armor.